### ISSUE TITLE
17 March Topic discussion

### ISSUE THREAD

**Issue by murraycadzow** [OtagoStudyGroup/studyGroup#15]
Leave comments about what lesson you would like to see/take next session

Offers to lead a session on a particular topic will be given priority

It doesn't have to be from the SYSKA topics, but we just had: Heatmaps, R+LaTeX, remote access (ssh), preprints, and inkscape if you would like to see more just say!

**Comment by johnfrombluff** [comment]
I'd like to learn about using git and github to contribute to an existing project, i.e.
- I've cloned a repo
- I've edited a file
- I've pushed my changes to my forked repo on Github
- What now? WTF is a "pull request"? How do I make one in a helpful way?

Maybe this is not an appropriate topic for SYSKA, and I should just RTFM?  I've actually started looking, but quickly became confused. Maybe I should try again?
